examples/CMakeLists.txt

```diff
@@ -41,10 +41,19 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #######################################################################
 
+set (chai_umpire_example_depends
+  chai)
+
+if (ENABLE_CUDA)
+  set (chai_umpire_example_depends
+    ${chai_umpire_example_depends{
```

Comment:
shouldn't this be:
` ${chai_umpire_example_depends}`?

Reply:
Should be fixed!